example/glue/MarkingDelegate.hpp

```diff
@@ -124,44 +172,31 @@ class MM_MarkingDelegate
 		GC_MixedObjectScanner *objectScanner = GC_MixedObjectScanner::newInstance(env, objectPtr, scannerSpace, 0);
 		*sizeToDo = sizeof(fomrobject_t) + objectScanner->getBytesRemaining();
 		return objectScanner;
-
 	}
 
 	/**
-	 * This method is called after the object graph depending from the root set has been traversed and all live
-	 * heap objects included in the root set and dependent graph have been marked. If there are other heap objects
-	 * that may not have been marked in the object reference graph traversal they should be marked here and, when
-	 * this additional marking is complete, MM_MarkingScheme::completeScan() must be called to mark live objects
-	 * depending from additional marked objects.
+	 * This method is called when a marked object cannot be enqueued on a work packet. This condition is handled
+	 * primariliy by OMR but such objects are presented here in case additional language-dependent handling is
+	 * required.
+	 *
+	 * This method is informational, no specific action is required.
 	 *
 	 * @param env The environment for the calling thread
 	 */
-	MMINLINE void completeMarking(MM_EnvironmentBase *env) { }
-
+	MMINLINE void handleWorkPacketOverflowItem(MM_EnvironmentBase *env, omrobjectptr_t objectPtr) { }
```

Comment:
If there is packet overflow in the example we will not handle it? What is this method supposed to do?